src/sentry/static/sentry/app/components/footer.tsx

```diff
@@ -56,6 +57,17 @@ const FooterLink = styled(ExternalLink)`
   }
 `;
 
+export const FooterIcon = styled(props => (
```

Comment:
```suggestion
const FooterIcon = styled(props => (
```

I think this is only used here..right?